ksqldb-common/src/main/java/io/confluent/ksql/schema/Operator.java

```diff
@@ -29,6 +29,10 @@
   ADD("+", SqlDecimal::add) {
     @Override
     public SqlType resultType(final SqlType left, final SqlType right) {
+      if (left == null || right == null) {
```

Comment:
does the interpreter code use this as well? we should make sure we have tests for this with the interpreter enabled and disabled (cc @AlanConfluent can point you how to do that if you're lost)

Reply:
the tests in `ExpressionEvaluatorParityTest.java` take care of that